tests/ui/suggest-null-ptr.rs

```diff
@@ -0,0 +1,22 @@
+// Suggest providing a std::ptr::null if provided with
```

Comment:
```suggestion
// run-rustfix

// Suggest providing a std::ptr::null if provided with
```
To make sure that the suggestion works as expected.